src/content/guides/ecommerce.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: eCommerce
+description: How to process payments using Centrapay's eCommerce Platform.
+nav:
+  path: Reference/Merchant Integrations
+  order: 2
+---
+
+Centrapay enables businesses to process payments for connected Centrapay assets online. In order to process payments businesses must be able to integrate with one of our eCommerce payment flows.
+
+## Onboarding
+
+In order to request eCommerce payments, integrators will need a [merchant](https://docs.centrapay.com/api/managed-integrations/#merchant-model), a [merchant configuration](https://docs.centrapay.com/api/merchant-configs/), and a merchant-terminal [API key](https://docs.centrapay.com/api/api-keys/).
+Centrapay will provision these on behalf of the integrators. Initially the merchant and merchant configuration you receive will be configured to accept test resources.
```

Comment:
"Centrapay will provision these on behalf of the integrators" - We may want to encourage integrators to create merchants and merchant configs through the business portal.

Reply:
Yeah maybe eventually I just didn't want to add anything that they can't currently do. I'm not sure what the evolution of e-commerce merchants might look like, I could see it being more locked down then creating physical merchants.